CollAction/.gitignore

```diff
@@ -1,4 +1,3 @@
 wwwroot/js/*.bundle.js
-wwwroot/js/bundle.js
```

Comment:
You removed this line and added bundle.js to the repository?

Reply:
I don't see bundle.js in the repository... shouldn't be there since it's not created anymore. But I'll put it back in gitignore just in case.